.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,9 @@
+blank_issues_enabled: false
+
+contact_links:
+  - name: Have you read the docs?
+    url: https://llama-stack.readthedocs.io/en/latest/index.html
+    about: Much help can be found in the docs
+  - name: Chat on Discord
```

Comment:
Thanks -- this makes sense.

I think adding a note about discussions would also be useful here. Because the first thing people should do is create a discussion. Only once we all decide something is actionable does it become an Issue.

Reply:
Sounds good. Added.